administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -2002,15 +2002,9 @@ public function child()
         }
 
         $user = $this->getCurrentUser();
-        unset($xml->languages);
-        unset($xml->media);
-        unset($xml->files);
-        unset($xml->parent);
-        unset($xml->inheritable);
 
         // Remove the update parts
-        unset($xml->update);
-        unset($xml->updateservers);
+        unset($xml->languages, $xml->media, $xml->files, $xml->parent, $xml->inheritable, $xml->update, $xml->updateservers);
```

Comment:
```suggestion
        unset(
            $xml->languages,
            $xml->media,
            $xml->files,
            $xml->parent,
            $xml->inheritable,
            $xml->update,
            $xml->updateservers
        );
```
line feels to long and hard to read